container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -4364,6 +4507,11 @@ variables:
   options:
     description: Concatenation of trusted CA certificates to be made available on
       the cell.
+- name: UAADB_PASSWORD
```

Comment:
Could you name it `UAA_DATABASE_PASSWORD` or `UAADB_DATABASE_PASSWORD` to match the other database passwords?

Reply:
I could, but then we should change it in the UAA chart as well: https://github.com/SUSE/uaa-fissile-release/blob/develop/container-host-files/etc/scf/config/role-manifest.yml#L560-L564

But then you have to worry about the upgrade scenario. So not really in favour of changing the name.